project/notification/factory.py

```diff
@@ -1,19 +1,15 @@
 import factory
 from . import models
+from shared_factory.shared_factory import FreezeTimeModelFactory
+from accounts.factory import ProfileFactory
+from threads.factory import ThreadFactory, CiviFactory
 
-class UserFactory(factory.Factory):
+class NotificationFactory(FreezeTimeModelFactory):
     class Meta:
-        model = models.User
+        model = models.Notification
 
-    first_name = 'John'
-    last_name = 'Doe'
-    admin = False
-
-# Another, different, factory for the same object
-class AdminFactory(factory.Factory):
-    class Meta:
-        model = models.User
-
-    first_name = 'Admin'
-    last_name = 'User'
-    admin = True
\ No newline at end of file
+    account = factory.SubFactory(ProfileFactory)
+    thread = factory.SubFactory(ThreadFactory)
+    civi = factory.SubFactory(CiviFactory)
+    activity_type = factory.fuzzy.FuzzyChoice(models.Notification.activity_CHOICES, getter=lambda c: c[0])
+    read = factory.Faker().pybool()
```

Comment:
Add new line at end of file
```suggestion
    read = factory.Faker().pybool()
    
```